### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by endersonmenezes**
This pull request introduces a new "Club" module to the backend, providing a comprehensive set of wallet and raffle management features, along with admin endpoints and thorough test coverage. It also improves authentication flows and updates documentation for community addition.

**Backend: Club module and wallet/raffle management**

* Added a new `ClubModule` with controller, service, and integration with TypeORM entities for wallets, wallet transactions, raffles, and raffle entries. This module supports both member and company wallets and transactions. (`backend/src/club/club.module.ts`, `backend/src/app.module.ts`) [[1]](diffhunk://#diff-f1b02ae7d46a8811b977752da568a6d5b6c89a3a5574fb012bd8228049c849caR1-R33) [[2]](diffhunk://#diff-0c2985c7b189bc65e8ded71e24c833e2d788288beaba00cad243a42ce5c625a2R17-R18) [[3]](diffhunk://#diff-0c2985c7b189bc65e8ded71e24c833e2d788288beaba00cad243a42ce5c625a2R56-R57)
* Implemented `ClubController` with endpoints for:
  - Member wallet management (view wallet, transactions)
  - Public wallet lookup by handle
  - Admin wallet adjustments and unified transaction history
  - Raffle operations: listing, entry, creation, drawing, and cancellation, with role-based access control (`backend/src/club/club.controller.ts`)
* Added comprehensive tests for `ClubController`, covering wallet, transaction, and raffle endpoints, including admin actions and edge cases (`backend/src/club/club.controller.spec.ts`)

**Authentication improvements**

* Centralized session cookie options to ensure `clearCookie` and `cookie` use identical attributes, preventing issues in browsers like Safari; now used in both login and logout flows (`backend/src/auth/auth.controller.ts`) [[1]](diffhunk://#diff-57dfc3198cff39a2a2c4bd2e457609fe83e5287ede4110e2fa98b00753137e32L5-R5) [[2]](diffhunk://#diff-57dfc3198cff39a2a2c4bd2e457609fe83e5287ede4110e2fa98b00753137e32R17-R31) [[3]](diffhunk://#diff-57dfc3198cff39a2a2c4bd2e457609fe83e5287ede4110e2fa98b00753137e32L154-L158) [[4]](diffhunk://#diff-57dfc3198cff39a2a2c4bd2e457609fe83e5287ede4110e2fa98b00753137e32L211-R224)
* Enhanced GitHub OAuth login to support account switching by forwarding the `login` query parameter as a login hint to GitHub, enabling explicit account selection or re-login (`backend/src/auth/github-auth.guard.ts`) [[1]](diffhunk://#diff-0ebda0be302698deec9a3a188d8bd53de72403cb47089a2f348a331a0eb8a5a9R21-R26) [[2]](diffhunk://#diff-0ebda0be302698deec9a3a188d8bd53de72403cb47089a2f348a331a0eb8a5a9R43-R56)

**Documentation and skills update**

* Added a new `community-add` skill for guiding the addition of partner communities, and updated the skills documentation accordingly (`.github/skills/community-add/SKILL.md`, `.github/agents/codaqui-specialist.agent.md`) [[1]](diffhunk://#diff-01b5f80f666191184b5a8fef6306cd49347cf5e4a240660e68d984f39957f137R2-L7) [[2]](diffhunk://#diff-01b5f80f666191184b5a8fef6306cd49347cf5e4a240660e68d984f39957f137L88-R88) [[3]](diffhunk://#diff-dfb158ca69a8f5d1859bf3c8c630d78f94355e9257b2d345e0056655ac79c747R205)

These changes collectively introduce a robust infrastructure for wallet and raffle features, improve authentication reliability and flexibility, and expand documentation and automation for community management.

### CODE REVIEW

File: backend/src/stripe/stripe.service.ts [hunk 1]
```diff
@@ -341,11 +378,136 @@ export class StripeService {
...
+        metadata: {
+          entityType: 'business',
+          companyId,
+          memberId,
+          isSubscription: 'true',
+        },
+      },
+      ...(company.stripeCustomerId ? { customer: company.stripeCustomerId } : {}),
+      ui_mode: 'embedded_page' as const,
+      return_url: returnUrl,
```
[the remark this answers is not in the document]
Comment (endersonmenezes) [reply backend/src/stripe/stripe.service.ts:435]: Obrigado pelo apontamento. Neste projeto usamos Stripe SDK 21, cujo tipo oficial `SessionCreateParams.UiMode` expõe `embedded_page` (não `embedded`) para Checkout Session. Trocar para `embedded` quebra a tipagem/compilação nesta stack, então mantive `embedded_page` por compatibilidade com a versão atual.
